data/json/monsters/reptile_amphibian.json

```diff
@@ -217,10 +217,16 @@
     "melee_dice": 5,
     "melee_dice_sides": 4,
     "dodge": 1,
-    "armor_bash": 30,
-    "armor_cut": 30,
-    "armor_bullet": 35,
-    "armor_stab": 30,
+    "armor_bash": 50,
```

Comment:
all armor and hp changes are attempts to compensate for the large and conspicuous weakpoint added below.